sqe-dtos/Polygon.cs

```diff
@@ -4,7 +4,10 @@ public class PolygonDTO
 	{
 		public string mask { get; set; }
 		public uint maskEditorId { get; set; }
-		public string transformMatrix { get; set; } //CAN BE NULL
-		public uint transformMatrixEditorId { get; set; }
+		public float? scale { get; set; } // Can be null
```

Comment:
Since scale, rotation and translation seem to be grouped together in numerous places, perhaps we should have a TransormationDTO that has a float? scale, float? rotation and OffsetDTO? translation . So if we happen to add another potential transform, we can add it there and have it everywhere. I wouldn't be surprised if a few months down the road we will add a matrix for the transform (applied after the scale, rotation and translation), for more complicated linear transformations.

Reply:
Yeah, I think that is a good idea.  And I hope we do not add a matrix on top of it.  By the way, I think we need to make sure that the transform operation is properly applied always in the standard SRT (scale, rotate, translate) order so that any GIS functions in the DB/API will match the front end presentation.  A different order of operations can result in a different result (which is one reason I liked the safety of the transform matrix).